dask/array/core.py

```diff
@@ -2918,8 +2918,12 @@ def to_zarr(
         If given array already exists, overwrite=False will cause an error,
         where overwrite=True will replace the existing data.  Note that this
         check is done at computation time, not during graph creation.
-    compute, return_stored: see ``store()``
-    kwargs: passed to the ``zarr.create()`` function, e.g., compression options
+    compute: bool
+        See ``store()``.
```

Comment:
I realize this predates this PR, but maybe we something like "See ``store()`` for more details." would be slightly more clear. Additionally, feel free to link directly to the `store` API docs here